enterprise/server/tasksize/tasksize.go

```diff
@@ -23,6 +23,12 @@ const (
 	DefaultCPUEstimate      = int64(600)
 	DefaultFreeDiskEstimate = int64(100 * 1e6) // 100 MB
 
+	// Additional resources needed depending on task characteristics
+
+	FirecrackerAdditionalMemEstimateBytes = int64(150 * 1e6) // 150 MB
+	DockerAdditionalMemEstimateBytes      = int64(800 * 1e6) // 800 MB
+	DockerAdditionalDiskEstimateBytes     = int64(12 * 1e9)  // 12 GB
```

Comment:
We should look into making the default free space we allocate for firecracker larger. Need to check if there are any perf implications. I think we have enough local SSD on the executors that we can afford to be generous with default size as long as there isn't significant performance overhead.